<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>AndryBlack</author>
 <class>FontBuilder</class>
 <widget class="QMainWindow" name="FontBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FontBuilder</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_Font">
       <attribute name="title">
        <string>Font</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_FontSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Font</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="FontSelectFrame" name="frameFontSelect" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="FontOptionsFrame" name="frameFontOptions" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Charaters">
       <attribute name="title">
        <string>Characters</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="CharactersFrame" name="frameCharacters" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Layout">
       <attribute name="title">
        <string>Layout</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Layout engine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLayouter"/>
        </item>
        <item>
         <widget class="LayoutConfigFrame" name="frameLayoutConfig" native="true"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Output">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="OutputFrame" name="frameOutput" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonWriteFont">
          <property name="text">
           <string>Write font</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_Preview">
       <attribute name="title">
        <string>Font image preview</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,0,0" columnminimumwidth="0,0,0,0">
        <item row="3" column="0" colspan="4">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>360</width>
             <height>425</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="4" column="0" colspan="4">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" colspan="4">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="FontDrawWidget" name="widgetFontPreview" native="true"/>
            </item>
            <item row="2" column="3">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBoxDrawGrid">
          <property name="text">
           <string>Draw grid</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_ImageSize">
          <property name="text">
           <string>0x0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>scale:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="comboBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>50%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800%</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_TestFont">
       <attribute name="title">
        <string>Test font</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="FontTestFrame" name="fontTestFrame" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FontSelectFrame</class>
   <extends>QWidget</extends>
   <header>fontselectframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontOptionsFrame</class>
   <extends>QWidget</extends>
   <header>fontoptionsframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharactersFrame</class>
   <extends>QWidget</extends>
   <header>charactersframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayoutConfigFrame</class>
   <extends>QWidget</extends>
   <header>layoutconfigframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OutputFrame</class>
   <extends>QWidget</extends>
   <header>outputframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontTestFrame</class>
   <extends>QWidget</extends>
   <header>fonttestframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontDrawWidget</class>
   <extends>QWidget</extends>
   <header>fontdrawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>fontParametersChanged()</slot>
  <slot>requestFontsDirChange()</slot>
 </slots>
</ui>
